client/rest-high-level/src/main/java/org/opensearch/client/TimedRequest.java

```diff
@@ -47,7 +47,7 @@
     public static final TimeValue DEFAULT_MASTER_NODE_TIMEOUT = TimeValue.timeValueSeconds(30);
 
     private TimeValue timeout = DEFAULT_ACK_TIMEOUT;
-    private TimeValue masterTimeout = DEFAULT_MASTER_NODE_TIMEOUT;
+    private TimeValue clusterManagerTimeout = DEFAULT_MASTER_NODE_TIMEOUT;
```

Comment:
Question: Is it better to name the variable as `clustermanagerTimeout` ?

Reply:
Answer: Since cluster manager is 2 words, I will keep using camel case in the variable names.